solution_runner/utils.py

```diff
@@ -0,0 +1,17 @@
+import importlib
+from pathlib import Path
+from types import ModuleType
+
+import consts
+
+
+def get_module_from_filepath(filepath: Path, directory_relative_to: Path | str = Path('.')) -> ModuleType:
```

Comment:
```suggestion
def get_module_from_filepath(filepath: Path, directory_relative_to=Path(os.getcwd())) -> ModuleType:
```
* `directory_relative_to` cannot be a string because you call `.absolute()` on it.
* I think it is nicer to explicitly refer to the current working directory.

Reply:
Thanks!
-Fixed in 5bb4704d.
- Fixed in f2ca4a0a.